src/instructlab/model/train.py

```diff
@@ -390,6 +399,12 @@ def clickpath_setup(is_dir: bool) -> click.Path:
     is_flag=True,
     help="By default, checkpoints are saved at the end of each training epoch. This option disables this behavior.",
 )
+@click.option(
+    "--training-journal",
+    type=click.Path(path_type=pathlib.Path),
+    cls=clickext.ConfigOption,
+    help="Path to journal file or the directory in which to save the journal file.",
```

Comment:
You don't need to write a help here, since we use `cls=clickext.ConfigOption`, the help will be be inherited from the Config Field description (in configuration.py). See https://github.com/instructlab/instructlab/pull/2201 for reference.

Reply:
++

Reply:
fixed